src/main/riddl/Venues/Reservations/reservationTypeDefinitions.riddl

```diff
@@ -0,0 +1,23 @@
+type Status is any of {Draft, OnHold, Reserved, Active, Past} briefly "Internal status of a Reservation."
+
+type Info is {
+    range: TimeRange,
+    title: String,
+    description: String,
+    forOrg: OrganizationId,
+    isPrivate: Boolean briefly "Is this reservation accessible outside of the organization owning it?"
```

Comment:
This brings up the question of 'what is the difference between Reservation and Event?'. I'm not exactly sure what you had in mind. When I was showing him my code yesterday, Reid was surprised that Reservation was even being included in this context. Is it mostly a financial administration thing or is there more to it?

Reply:
A Reservation is simply a contract to be able to use a particular place at a particular time. As long as it is not illegal and it fits with the terms of use established by the Venue/Location, I am not sure that the "owner" cares what happens. I think eventually there could be a financial aspect to a Reservation. You pay for use. If the reservation is a contract to use, the Event is what is happening during that time. A play, a game, a class, a working session, etc. People get invited to an event not a reservation. Does that help clear things up?

Reply:
So should we be including legal aspects like Terms of Use in our Venue, Location, and Reservation definitions right now? How will that shape out - because I feel like it is better, but more difficult maybe, to provide a way to restrict who books and what they can do through the app, rather than than build out some some of automatic notification mechanism for indicators of illegality

Reply:
No. Future scope for sure.